components/authentication/tenants/shebert.yaml

```diff
@@ -10,3 +10,80 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: admin
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: admin-managed-shebert
+  namespace: managed-shebert
+subjects:
+  - kind: User
+    name: scoheb
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: admin
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: create-rs-scoheb
+  namespace: managed-shebert
+subjects:
+  - kind: User
+    name: scoheb
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: release-service-releasestrategy-editor-role
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gitopsdeployment-manager
```

Comment:
ran a search and found we already have something similar - do you want to reconcile these into one?

https://github.com/redhat-appstudio/infra-deployments/blob/9da0f0b67256c53171994a66807cbcce7769119b/components/authentication/tenants/shbose.yaml#L43

Reply:
sure I can reconcile. Any suggestion as to where this can be extracted to?

Reply:
Tricky :) How about updating the one that already exists ?